install-photobooth.sh

```diff
@@ -499,6 +499,10 @@ function common_software() {
     if [ "$NEEDS_NPM_CHECK" = true ]; then
         check_npm
     fi
+
+    # Install php composer
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

Comment:
composer exists inside the bin/ directory of Photobooth

https://github.com/PhotoboothProject/photobooth/blob/dev/bin/composer

Reply:
Ok, thought I also got an error during `npm run build`